bootstrap.sh

```diff
@@ -400,6 +400,9 @@ setup_packages() {
         return 1
     fi
 
+    info "Update sources"
+    sudo apt-get update
```

Comment:
If we do this here at this point, the script would no longer work on Linux systems that do not use apt. Can you maybe make an `update_sources` function, call it here and then check in it like `prompt_install` the used package manager to update the sources?